### ISSUE TITLE
place_limit_buy_order fails with no error when ask_price is a variable

### ISSUE THREAD

**Issue by benkelley1979** [robinhood-unofficial/pyrh#249]
Win10 / Python 3.8.2 / working in IDLE x64

TL;DR:
I can make `place_limit_buy_order` work from IDLE console when I enter `ask_price` manually as a float value.
The `place_limit_buy_order` command fails once I use a variable for `ask_price`
I encountered similar issues with `place_buy_order`

`float(quote.get('last_trade_price'))`
> 1.7502

`type(float(journal.Price[len(journal.index)-1]))`
> <class 'float'>

`rh.place_limit_buy_order('https://api.robinhood.com/instruments/**GUID**/', 'XTNT', 'GFD',  float(journal.Price[len(journal.index)-1]), 5)`
> _no response at all_

`rh.place_limit_buy_order('https://api.robinhood.com/instruments/**GUID**/', 'XTNT', 'GFD', 1.70, 5)`
> <Response [201]>

My desired result is to pull `instrument_URL ` `symbol` `price` `quantity` from lists. Nothing else seems to cause an issue except for `price`

lmk if you need clarification or would like to see my entire code.

**Comment by davidwang20** [comment]
also having this issue, it is very strange

**Comment by davidwang20** [comment]
Actually, I think i figured out what it is. Try truncating your float variable to 2 decimal places. That fixed it for me

**Comment by benkelley1979** [comment]
Hi @davidwang20 I had actually stumbled onto this earlier today and hadn't updated this post yet. I'm glad you got it going too!
I would like to understand if this is a limitation of pyrh or of Robinhood's API. When using the mobile app or website I can set orders to 4 decimal places. If possible I would like that option.
Also, shouldn't there be an error returned? I am not importing pyrh.Exceptions. Could that be the issue?

**Comment by benkelley1979** [comment]
> Actually, I think i figured out what it is. Try truncating your float variable to 2 decimal places. That fixed it for me

I found it helpful to create round_down and round_up functions. round_down to 2 decimals for limit sell and round_up to 2d for limit buy. This gives you the best chance at executing immediately
